### ISSUE TITLE
How does deployment work

### ISSUE THREAD

**Issue by BinderDavid** [haskell/cabal-website#39]
The `README.md` says that the site is deployed from the `gh-pages` branch, but that does not seem to be correct anymore. The last time that branch was updated from a travis CI action was 4 years ago, but the published website contains changes that are newer than that. The site does not seem to be deployed automatically, because https://www.haskell.org/cabal/ does not contain the changes from https://github.com/haskell/cabal-website/pull/38

**Comment by BinderDavid** [comment]
Ah, I could already answer this myself: https://github.com/haskell/cabal-website/issues/5#issuecomment-1873766247
It is deployed manually. The issue came up here: https://discourse.haskell.org/t/ghc-and-cabal-the-big-picture/9968/20?u=davidb

**Comment by ulysses4ever** [comment]
Indeed, thanks for the ping!

@Mikolaj it'd be great to deploy the current version that fixed horribly outdated instructions (Haskell Platform etc.)

**Comment by Mikolaj** [comment]
Indeed, let's deploy. I remember that was a part of the release checklist. Did it vanish in the last release checklist purge? Or is the deployment process described elsewhere?

**Comment by BinderDavid** [comment]
I also discovered that the `gh-pages` branch is currently used to host the haddocks of `Cabal-3.3`, which is still reachable under https://haskell.github.io/cabal-website/doc/html/Cabal/ That doesn't seem to be very useful, since `Cabal-3.3` is only a development version that was never released. Maybe we should just disable the `github pages` feature for this repo and delete the `gh-pages` branch as well?

**Comment by ulysses4ever** [comment]
Good catch. Should probably be disabled. @Mikolaj, @Kleidukos?

**Comment by Mikolaj** [comment]
Yes, let's gut it!

**Comment by Mikolaj** [comment]
Re deployment, it's described in https://github.com/haskell/cabal/wiki/Making-a-release#c6-publishing-everything-else,  and I just did it and it works. Let me know if this needs an improvement or if we can/should limit the bus factor --- I have no idea who beside me has the right --- all haskell org members maybe (I don't think I'm a member so I must have gotten it from Gershom or somebody during my first releases/deployments)?
